packages/browser/__tests__/sessionInfo/SessionInfoManager.spec.ts

```diff
@@ -96,4 +104,59 @@ describe("SessionInfoManager", () => {
       expect(session).toBeUndefined();
     });
   });
+
+  describe("clear", () => {
+    it("clears oidc data", async () => {
+      const sessionManager = getSessionInfoManager({
+        storageUtility: mockStorageUtility({}, true),
+      });
+      await sessionManager.clear("mySession");
```

Comment:
I don't think the value of the session param matters for this test, so I'd suggest:
```suggestion
      await sessionManager.clear("Value of sessionId doesn't matter");
```